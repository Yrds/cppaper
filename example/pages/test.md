# Test

Testing cmark parsers

# Testing unsafe HTML (should accept)

<script>
    console.log("hello world");
</script>

{# secret #}

~~strikethrough~~

| foo | bar |
| --- | --- |
| baz | bim |
