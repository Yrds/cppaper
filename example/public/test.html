<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width" />
    <title>Templates</title>
  </head>
  <body>
    <h1>Test</h1>
<p>Testing cmark parsers</p>

  </body>
</html>
