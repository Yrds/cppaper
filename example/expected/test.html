<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width" />
    <title>Yuri Santos - Test Page</title>
  </head>
  <body>
    <h1>Test</h1>
<p>Testing cmark parsers</p>
<h1>Testing unsafe HTML (should accept)</h1>
<script>
    console.log("hello world");
</script>
<p><del>strikethrough</del></p>
<table>
<thead>
<tr>
<th>foo</th>
<th>bar</th>
</tr>
</thead>
<tbody>
<tr>
<td>baz</td>
<td>bim</td>
</tr>
</tbody>
</table>

  </body>
</html>
